incubator/hnc/third_party/open-policy-agent/gatekeeper/pkg/webhook/certs.go

```diff
@@ -503,20 +538,50 @@ func (r *ReconcileVWH) Reconcile(request reconcile.Request) (reconcile.Result, e
 		return reconcile.Result{Requeue: true}, err
 	}
 
+	crds := []*unstructured.Unstructured{}
```

Comment:
What do we gain from using `Unstructured` here as opposed to the real CRD type? We know exactly what type it is and we manipulate its fields directly. (I have the same question for VWH but that's out of scope for this change)

Reply:
Maybe it's easier to make changes to the version that we just need to update it in one place in the global variable `vwhGVK` instead of updating all of them in the code?

Reply:
I'm not sure I follow? I'd suggest trying to use the real types for your new code and seeing if you like it. We can worry about the existing code later.

Reply:
Oh even in the new code, I refactored the cert injection to take the current `injectCert` functions as a parameter, and update the unstructed object accordingly (either validatingwebhookconfiguration or CRD).

Reply:
Ok why don't you leave this as-is.